### ISSUE TITLE
about the pick of standard deviation in Gaussian distribution

### ISSUE THREAD

**Issue by BaiMeiyingxue** [nzl-thu/MUSDL#6]
Hi Zanlin Ni,
Thanks for your wonderful work!
I was wondering whether the selection of standard deviation has an effect on the loss(_Here σ is a hyper-parameter which serves as the level of uncertainty for assessing an action_) since if I pick a bigger one, the loss in the training phase is close to zero, and if I choose 1, the loss is about 0.01. Will the size of the loss value have an impact on training?
Looking forward to your reply!
